### ISSUE TITLE
translateX and translateY should accept Size Percentage as well as Size LengthUnit argument

### ISSUE THREAD

**Issue by ddssff** [sebastiaanvisser/clay#218]
Currently I have to say 
```
transform $ translateX (other "-50%")
transform $ translateY (other "-50%")
```
On the other hand, `transform $ translate (pct (- 50)) (pct (- 50))` works and is what you actually want, the two transforms above interfere with each other.

**Comment by ddssff** [comment]
`translate` can handle this btw: 
```
> renderWith compact [] (transform (translate (pct 50) (pct 50)))
"{-webkit-transform:translate(50%,50%);-moz-transform:translate(50%,50%);-ms-transform:translate(50%,50%);-o-transform:translate(50%,50%);transform:translate(50%,50%)}"
```

**Comment by turion** [comment]
> Currently I have to say
> 
> ```
> transform $ translateX (other "-50%")
> transform $ translateY (other "-50%")
> ```

Instead of what do you have to say this?

**Comment by ddssff** [comment]
I'm not sure I understand the question.

**Comment by turion** [comment]
You wrote "Currently I have to say", but I'm not sure what the problem with the code `transform $ translateX (other "-50%")` is. Is the problem that you have to use `other` instead of `pct`?

**Comment by ddssff** [comment]
Yes that's it.  `other` is unsafe.

**Comment by ddssff** [comment]
Closing issues because they are old makes no sense to me.

**Comment by turion** [comment]
Fair enough. It's been closed though because there wasn't any activity for over a year. I'll assume you want this issue to stay open.

What we need to drive the issue forward is a concrete proposal how to change the API (bonus points for a sketch how to change the implementation).
